test/WebJobs.Extensions.Tests/PublicSurfaceTests.cs

```diff
@@ -26,13 +26,11 @@ public void ExtensionsPublicSurface_LimitedToSpecificTypes()
                 "FilesOptions",
                 "FilesWebJobsBuilderExtensions",
                 "FileSystemScheduleMonitor",
-                "StorageScheduleMonitor",
                 "FileTriggerAttribute",
                 "IFileProcessorFactory",
                 "ScheduleMonitor",
                 "ScheduleStatus",
                 "TimerInfo",
-                "TimerWebJobsBuilderExtensions",
```

Comment:
This should remain as noted above. For your new assembly, you'll want to add a new set of public surface tests